### PR TITLE
Link Feedback Form

### LINKED ISSUE #20: Feedback Form
### **Describe the bug:**
No feed back form is available but maintained at the footer, I want to make that form.
InkedCS Hut - Personal - Microsoft​ Edge 6_10_2021 08_07_31 PM_LI

### PR DESCRIPTION
**Pull request by rishabhrathore055**
# Description

Please include a summary of the change and which issue is fixed. List any dependencies that are required for this change.

Fixed #20 

## Type of change

Please check options that are relevant to your PR.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] This change requires a documentation update

# Checklist:

- [x] I have pulled the latest changes in the repository
- [x] I have squashed my commits
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/HomePage/Footer.js [hunk 1]
```diff
@@ -15,3 +15,23 @@ function Footer() {
 }
```
Comment (shriya030) [inline src/HomePage/Footer.js]: Kindly remove the repeated code.

**Review by shriya030 — CHANGES_REQUESTED** [review]
Kindly remove the repeated code

**Comment by rishabhrathore055** [comment]
@shriya030 pls review it

**Review by shriya030 — APPROVED** [review]

**Review by aj3011 — APPROVED** [review]

**Comment by rishabhrathore055** [comment]
@shriya030 please assign levels

**Pull request merged**